schema/index.js

```diff
@@ -101,7 +102,8 @@ const schema = new GraphQLSchema({
     fields: {
       followArtist: FollowArtist,
       updateCollectorProfile: UpdateCollectorProfile,
-      updateConversation: UpdateConversation,
+      updateBuyerOutcome: UpdateBuyerOutcome,
+      appendConversationThread: AppendConversationThread,
```

Comment:
I think naming the mutation `updateConversation` makes more sense than `appendConversationThread`.
In terms of grouping multiple mutating operations, although we only have one for now.

In the future, we might have to add other mutations related to a conversation (ex: deleting a message).
Then we can add an input field called `operation` with values like `add` or `remove`.